main.go

```diff
@@ -0,0 +1,56 @@
+// Шаг 1 ТЗ - в main.go обработчики + инициализация
+// Шаг 2 ТЗ - в db.go все про БД + func GetTaskByID, func UpdateTask, func DeleteTaskByID
+// Шаг 3-4 ТЗ - в nextdate.go func NextDate с правилами повторения + func AddTaskHandler добавление задач
+// Шаг 5 ТЗ - в task.go func GetTasksHandler получение задач
+// Шаг 6 ТЗ - в getPut.go func TaskHandler(получения и обновления задач) + func getTask(по идентификатору) + func updateTask
+// Шаг 7 ТЗ - в doneDel func MarkTaskDone отметка о выполнении + func DeleteTask удаление
+// все тесты проходят
+
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"go_final_project/api"
```

Comment:
Пришлось поиграться с именами пакетов, чтобы все запустить, конечно
```suggestion
	"github.com/OxanaSalchak/go_final_project/api"
```